waves/project.py

```diff
@@ -1229,10 +1369,11 @@ def capacity(self, units: str = "mw") -> float:
         raise ValueError("`units` must be one of: 'kw', 'mw', or 'gw'.")
 
     def capex(
-        self, breakdown: bool = False, per_capacity: str | None = None
+        self,
```

Comment:
This is more of a comment on my own design, but I think the kwargs in this method should be prefaced with a "*", like `Project.capex(self, *, breakdown, per_capacity):` to avoid accidental misuse of boolean flags. This would also apply to the `capex_orbit` and `capex_landbosse` methods, which should receive `fn(breakdown=breakdown)` instead of `fn(breakdown)`.